packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -397,8 +429,15 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
    */
   protected async closeDownloadingChannels(torrent: PaidStreamingTorrent) {
     torrent.wires.forEach(async wire => {
-      if (wire.paidStreamingExtension && wire.paidStreamingExtension.peerChannelId) {
-        await this.paymentChannelClient.closeChannel(wire.paidStreamingExtension.peerChannelId);
+      const {peerChannelId} = wire.paidStreamingExtension;
+      if (peerChannelId) {
+        log.info(`About to close channel ${peerChannelId}`);
+        wire.paidStreamingExtension.peerChannelId = null;
```

Comment:
with this one, we clean up the downloading channels when pausing, to ensure that the peer won't be able to pay if it tries.
It also removes an unnecessary check and adds a couple of extra logs that I think are usefull